make_release.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+#
+# Prepare ZIP files for upload as release files on Github.
+#
+# Before running this script,
+# you need to compile the binaries for all platforms
+# and store them in a local `release/` folder:
+#
+# $ tree release
+# release/
+# ├── include
+# │   └── smileapi
+# │       └── SMILEapi.h
+# ├── licenses
+# │   ├── LibSVM.txt
+# │   ├── LICENSE
+# │   ├── newmat.txt
+# │   ├── Rapidjson.txt
+# │   └── Speex.txt
+# ├── linux-armv7
+# │   ├── bin
+# │   │   └── SMILExtract
+# │   └── lib
+# │       └── libSMILEapi.so
+# ├── linux-armv8
+# │   ├── bin
+# │   │   └── SMILExtract
+# │   └── lib
+# │       └── libSMILEapi.so
+# ├── linux-x86_64
+# │   ├── bin
+# │   │   └── SMILExtract
+# │   └── lib
+# │       └── libSMILEapi.so
+# ├── macos-armv8
+# │   ├── bin
+# │   │   └── SMILExtract
+# │   └── lib
+# │       └── libSMILEapi.dylib
+# ├── macos-x86_64
+# │   ├── bin
+# │   │   └── SMILExtract
+# │   └── lib
+# │       └── libSMILEapi.dylib
+# └── windows-x86_64
+#     ├── bin
+#     │   └── SMILExtract.exe
+#     └── lib
+#         ├── SMILEapi.dll
+#         └── SMILEapi.lib
+# 
+# 21 directories, 19 files
+#
+# Those binaries can be build automatically
+# using our internal CI pipeline at
+# https://gitlab.audeering.com/tools/opensmile-ci.
+# Just make sure to first increase the version inside `conanfile.py`
+# and commit those changes as release,
+# and run the CI pipeline on that commit.
+
+# === Get version number ===
+# Extract version from conanfile,
+# see https://stackoverflow.com/a/43644495
+version=$(grep -Po '\bversion\s*=\s*"\K.*?(?=")' conanfile.py)
+
+# === Prepare docs ===
+# Create virtual environment,
+# build documentaton
+# and store inside the downloaded release folder.
+rm -rf venv
+virtualenv -p python3.10 venv
```

Comment:
Is Python 3.10 needed? Otherwise I wouldn't specify it here explicitly.

Reply:
Yes, as `requirements.txt.lock` is only valid for a given Python version. It might work with other versions, but you never know.

Reply:
Could we then use a lower version? not everyone might have 3.10 already.

Reply:
We are migrating our default Python version from 3.8 to 3.10 this month. So everyone should have Python 3.10 very soon.

Reply:
OK, that is interesting, thanks.